packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx

```diff
@@ -271,105 +265,45 @@ const ProofOfAddressForm = observer(
         } else {
             form_initial_values.address_state = '';
         }
-        const setOffset = (status: { msg: string }) => {
-            const mobile_scroll_offset = status?.msg ? '200px' : '154px';
-            return !isDesktop && !is_for_cfd_modal ? mobile_scroll_offset : '80px';
+        const setOffset = (status: { msg: string }): string => {
+            const mobile_scroll_offset = status?.msg ? '200px' : '166px';
+            return !isDesktop && !is_for_cfd_modal ? mobile_scroll_offset : '94px';
         };
 
         return (
             <Formik
                 initialValues={form_initial_values}
+                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                // @ts-ignore
                 onSubmit={onSubmitValues}
                 validate={validateFields}
                 enableReinitialize
             >
-                {({ status, handleSubmit, isSubmitting, isValid }) => (
-                    <>
+                {() => (
+                    <React.Fragment>
                         <LeaveConfirm onDirty={!isDesktop ? showForm : undefined} />
-                        {form_state.should_show_form && (
-                            <form noValidate className='account-form account-form_poa' onSubmit={handleSubmit}>
-                                <ThemedScrollbars
-                                    height='572px'
-                                    is_bypassed={!is_for_cfd_modal || !isDesktop}
+                        {form_state.should_show_form &&
+                            (isMobile ? (
```

Comment:
can we just not use ( !isDesktop ? condition 1 : condition 2 ) ?

Reply:
The layout is same for Desktop and Tablet. Hence used mobile